homeassistant/components/pioneer/const.py

```diff
@@ -0,0 +1,77 @@
+"""Consts used by pioneer."""
+from typing import Final
+
+DEFAULT_ZONE = 1
+
+CONF_SOURCES = "sources"
+CONF_ZONES = "zones"
+DOMAIN = "pioneer"
+DEFAULT_NAME = "Pioneer AVR"
+DEFAULT_PORT = 23  # telnet default. Some Pioneer AVRs use 8102
+DEFAULT_TIMEOUT: Final = 500
+DEFAULT_SOURCES: dict[str, str] = {
+    "Phono": "00",
+    "CD": "01",
+    "Tuner": "02",
+    "CD-R/Tape": "03",
+    "DVD": "04",
+    "TV/Sat": "05",
+    "Sat/Cbl": "06",
+    "Video 1": "10",
+    "Multi Channel In": "12",
+    "Video 2": "14",
+    "DVR/BDR": "15",
+    "iPod/USB": "17",
+    "XM Radio": "18",
+    "HDMI 1": "19",
+    "HDMI 2": "20",
+    "HDMI 3": "21",
+    "HDMI 4": "22",
+    "HDMI 5": "23",
+    "HDMI 6": "24",
+    "Blu-Ray": "25",
+    "Home Media Gallery (Internet Radio)": "26",
+    "Sirius": "27",
+    "Adapter Port": "33",
+    "Netradio": "38",
+    "Media Server": "44",
+    "Favorites": "45",
+    "Game": "49",
+}
+
+MAX_VOLUME = 185
+MAX_SOURCE_NUMBERS = 60
+
+
+ZONE_COMMANDS: dict[int, dict[str, dict[str, str]]] = {
```

Comment:
Please extract the device specific interface code and data parsing to a standalone library published on PyPI.

https://developers.home-assistant.io/docs/creating_component_code_review#4-communication-with-devicesservices